src/Alert/Alert.js

```diff
@@ -80,9 +82,18 @@ Alert.propTypes = {
     link: PropTypes.string,
     linkProps: PropTypes.object,
     linkText: PropTypes.string,
+    localizedText: CustomPropTypes.i18n({
+        closeButton: PropTypes.string
+    }),
     type: PropTypes.oneOf(ALERT_TYPES)
 };
 
+Alert.defaultProps = {
+    localizedText: {
+        closeButton: 'Close'
+    }
+};
+
 Alert.propDescriptions = {
     dismissible: 'Set to **true** to show a dismiss button.',
     link: 'Value to be applied to the anchor\'s `href` attribute.',
```

Comment:
are localizedText magic props that don't need descriptions because they go to their own table?